bukkit/pom.xml

```diff
@@ -216,7 +216,33 @@
         <!-- Bedrock player bridge -->
         <dependency>
             <groupId>org.geysermc</groupId>
-            <artifactId>connector</artifactId>
+            <artifactId>core</artifactId>
+            <version>${geyser.version}</version>
+            <scope>provided</scope>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <!-- If you remove these, build will fail -->
+        <dependency>
+            <groupId>org.geysermc</groupId>
+            <artifactId>geyser-api</artifactId>
```

Comment:
If the dependency is necessary, it shouldn't be excluded above. This exclusion rule exists to to prevent pulling in the other dependencies like jackson.

Reply:
Ok, so how should this be addressed? Core must be included, because that contains the ˛`AuthType` class. However, core is dependent on jackson. I have force pushed some changes? Is it OK?

Reply:
>Core must be included, because that contains the ˛AuthType class.

We could use multiple exclusion rules, so that the core dependency is still included, but not the other ones.

Reply:
But then every time Geyser adds a new dependency, we'll have to exclude it manually in FastLogin.

Anyways, I've looked into that and the list of excluded dependencies from Geyser would be longer (if we exclude one per line) than just adding `geyser-api` explicitly. [[There](https://github.com/GeyserMC/Geyser/blob/2953ea951321069a43b5b9195f960654b477ed73/core/pom.xml#L39-L254) are more than 200 lines of unneeded dependencies in Geyser's pom.xml].

Reply:
Good point